Formula/envoy.rb

```diff
@@ -7,6 +7,8 @@ class Envoy < Formula
       tag:      "v1.20.1",
       revision: "ea23f47b27464794980c05ab290a3b73d801405e"
   license "Apache-2.0"
+  revision 1
```

Comment:
```suggestion
```
Users don't need to reinstall Envoy after this change, so we should omit the revision.